README.md

```diff
@@ -77,102 +73,103 @@ Or in order to install to another root directory:
 
-Besides the default conf file, it's also possible to use a directory in where to
-put some extra configuration files. Files in this directory will be read in
-alphabetical order, and can add or override configurations found in previous
-files.
 
-By default, `/etc/mavlink-router/config.d` is the directory, but it can be
-overriden via a `MAVLINK_ROUTER_CONF_DIR` environment variable, or via the `-d`
-switch when running mavlink-routerd.
+To route mavlink packets from UART `ttyS1` to 2 other UDP endpoints, do as following:
 
-Suppose that the default configuration file defines an `UartEndpoint bravo` using `Baud=56600`,
-an example of overriding configuration would be:
+    $ mavlink-routerd -e 192.168.7.1:14550 -e 127.0.0.1:14550 /dev/ttyS1:1500000
+
+The `1500000` after the colon in `/dev/ttyS1:1500000` sets the UART baudrate.
+See more options with `mavlink-routerd --help`.
 
-```ini
-[UartEndpoint bravo]
-Baud = 115200
-```
+It's also possible to route mavlinks packets from an incoming UDP connection instead of UART:
+
+    $ mavlink-routerd -e 192.168.7.1:14550 -e 127.0.0.1:14550  0.0.0.0:24550
 
-That would change `Endpoint bravo` baudrate to `115200`.
+Additionally, mavlink-router also listens on port 5760 for TCP connections by default.
+Any client connecting to that port will be able to send and receive MAVLink data.
 
-### Flight stack logging ###
+IPv6 addresses must be enclosed in square brackets like this: `[::1]`.
+The port number can be specified in the same way, as with IPv4 then: `[::1]:14550`.
+Both unicast and multicast addresses should be handles properly and the interface for a link-local address is auto-detected.
 
-Mavlink router can also collect flight stack logs. It supports collecting
-both PX4 and Ardupilot flight stacks logs. To start logging, set a
-directory to the `Log` key in the `General` section in the config file (or use
-argument option `-l`).
-Currently, mavlink router needs to be informed which MAVLink dialect the
-flight stack speaks, `common` or `ardupilotmega`. To define it, use the
-`MavlinkDialect` key. For instance, to collect Ardupilot logs to
-`/var/log/flight-stack` directory, one could add to the conf file:
+
+## Detailed Description of Capabilities
+
+Here are some more detailes on the additional capabilities of mavlink-router.
+
+### Endpoint Configuration
+
+A description of the different endpoint operating modes and their configuration parameters can be found in the [examples/config.sample](examples/config.sample)
+
+### Flight Stack Logging
+
+Mavlink router can also collect flight stack logs.
+It supports collecting both PX4 and Ardupilot flight stacks logs.
+To start logging, set a directory to the `Log` key in the `General` section in the config file (or use argument option `-l`).
+Currently, mavlink router needs to be informed which MAVLink dialect the flight stack speaks, `common` or `ardupilotmega`.
```

Comment:
this is not longer true... there is an "auto" option.

Reply:
Sentence reworked